[ADD] Recruitment: New refuse applicants doc

larm-odoo opened the pull request:
Original Recruitment doc too long, breaking off the refusing applicants section into its own doc.

[Task Card](https://www.odoo.com/odoo/project/3835/tasks/3974771?cids=3) for this PR.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/9663).

larm-odoo commented:
Hi @Felicious - tagging you for a first-round, as we discussed, for this second doc in my recruitment breakaway docs.

Felicious commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,137 @@
+Refuse applicants
+=================
+
+At any point in the recruitment process, an applicant can be refused for a job position. To refuse
+an applicant, click on an applicant's card to navigate to a detailed view of the card. At the top of
```
Since this is a new doc, I included some detailed navigational instructions on where the user should go (: Feel free to alter the instructions! I just took a guess on where they should go; as the HR expert, you would know better :blush: 
```suggestion
At any point in the recruitment process, an applicant can be refused for a job position. To refuse
an applicant, start by going to the :menuselection:`Recruitment app --> Applications --> All Applications`. On the :guilabel:`Applications` list, click the (undesired) applicant. At the top of
```

larm-odoo replied:
Great idea- I added the other way to get to there (since there are two).

Felicious commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,137 @@
+Refuse reasons
+==============
+
+A :guilabel:`Refuse Reason` pop-up window appears. Click the desired :guilabel:`Refuse Reason` from
```
For skimmablility, I added a bit more context (:
```suggestion
Clicking :guilabel:`Refuse` makes the :guilabel:`Refuse Reason` pop-up window appear. Click the desired :guilabel:`Refuse Reason` from
```

Felicious commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,137 @@
+   dynamic content, the applicant's name appears anytime that dynamic placeholder appears on the
+   email template. For more detailed information on email templates, refer to the
+   :doc:`../../general/companies/email_template` documentation.
+
```
To complete the basic flow in this topmost section, I'm suggesting to bring the action to click the refuse button all the way up here, with the image. 

that way, readers who are just coming to the doc to see how applicants are refused can see the workflow right from the beginning, and additional features, such as configuring refuse reasons and the refusal email template, are found in later sections.

But, this is just a suggestion! Feel free to just ignore this comment, or reach out to me and we can have a discussion about it (:

```suggestion

After selecting a refuse reason, click the :guilabel:`Refuse` button to close the application.

.. image:: refuse_applicant/refuse.png
   :align: center
   :alt: An applicant's card with the refused banner appearing in the top-right corner in red.
```

Felicious commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,137 @@
+Create or modify refuse reasons
+-------------------------------
+
+To view all currently configured refuse reasons, navigate to :menuselection:`Recruitment app -->
```
As a reader, I'm slightly confused because I'm still curious about one thing: why should I create a specific refuse reason? As a hiring manager, how does it improve my hiring process? I see that in the reporting, you can filter by refuse reason, so is the refuse reason

Instead, I'd modify the first sentence to quickly define what a refuse reason is, like:

```suggestion
Refuse reasons allow multiple recruiters to consistently document why a candidate was not a good fit, and tailor specific refusal reason email templates. To configure refuse reasons, navigate to :menuselection:`Recruitment app -->
```

P.S. this is my interpretation of what I *think* a refuse reason is. Feel free to alter it as you see fit!

Felicious commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,137 @@
+-------------------------------
+
+To view all currently configured refuse reasons, navigate to :menuselection:`Recruitment app -->
+Settings --> Applications: Refuse Reasons`. All the refuse reasons appear in a list view.
```
```suggestion
Configuration --> Applications: Refuse Reasons`. All the refuse reasons appear in a list view.
```

Felicious commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,137 @@
+:guilabel:`Refuse Reasons` list. The new template appears in the new refuse reason :guilabel:`Email
+Template` field.
+
+To make any modifications to a refuse reason, click on the line, and make any desired changes.
```
```suggestion
To make modifications to an existing refuse reason, click the line, and make any desired changes.
```

Felicious commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,137 @@
+Send refusal email
+==================
+
+After a :ref:`Refuse Reason <recruitment/refuse-reasons>` is selected, two fields appear below the
```
For skimmability, I wanted to include more details for the reader about where we are in the workflow
```suggestion
After clicking the :guilabel:`Refuse` button on an applicant form, a :ref:`Refuse Reason <recruitment/refuse-reasons>` can be selected. Then, two fields appear below the
```

Felicious commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,137 @@
+Job position view
+-----------------
+
+To view only the refused applicants for that job position, click the :icon:`fa-caret-down`
```
how do we navigate to the job position's kanban view, again? :thinking:

Felicious reviewed (approved):
Great job on this straightforward doc, @larm-odoo :smile: 

I had just had very minor suggestions to add some more context at the beginning of each section for skim-ability, but other than that, incredible doc!

larm-odoo commented:
Thank you SO much @Felicious - your suggestions were great! I took them all and expanded a few where it made sense to do so. I also added a link to the email section, since I added a line about how an email i ssent after you refuse an applicant.

larm-odoo commented:
This is ready for a final review, @ksc-odoo!

ksc-odoo reviewed (approved):
Hey @larm-odoo -- just finished my Final Review. Great work. There are a decent amount of comments/feedback that require your attention, but I don't anticipate you having any issue addressing them, so I am approving now. You'll notice that the majority of them fall into the Optional/Suggestion category, so please implement the ones you deem necessary. If you have any questions about anything, don't hesitate to reach out. Anyway, once you implement the necessary changes/adjustments, feel free to tag this for Tech Review. Thanks! 👍

larm-odoo commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,163 @@
+
+.. _recruitment/email:
+
+Send refusal email
```
I thought about that- but I did adda sentence after the seciton you mentioned, and there is a link in the text to jump to the send email section. So because of that, I think it's ok- they can go in order (just with a click).

larm-odoo commented on content/applications/hr/recruitment/refuse_applicant.rst:
```diff
@@ -0,0 +1,163 @@
+Then, click :guilabel:`Create and edit...` and a :guilabel:`Create Email Template` form pop-up
+window appears.
+
+Enter a :guilabel:`Name` for the form and an email :guilabel:`Subject` in the corresponding fields.
```
I don't see that field- I just see the fields in the image in the doc. I checked other runbots as well- I don't see that anywhere.

larm-odoo commented:
Hi @samueljlieber - thisis ready for a tech review =)

samueljlieber reviewed (approved):
Hi @larm-odoo! Thanks for your work on this Recruitment PR. I agree that moving the Refusal reasons content to it's own doc is a good idea. 

While only the suggestions about RST syntax and custom anchor definitions are required, I do have some suggestions for you to improve the overall flow of this document, as I feel some instructions could be restructured or simplified.

Approving and delegating 👍
.....
@robodoo delegate=larm-odoo

larm-odoo commented:
@robodoo r+
